src/main/java/eu/arrowhead/client/library/ArrowheadService.java

```diff
@@ -407,6 +409,111 @@ public OrchestrationResponseDTO queryOrchestrationStore(final long systemId) {
 		return response.getBody();
 	}
 	
+	//-------------------------------------------------------------------------------------------------
+	/**
+	 * Sends a http(s) 'subscription' request to Event Handler Core System.
+	 * 
+	 * @param request SubscriptionRequestDTO which represents the required payload of the http(s) request
+	 * @throws AuthException when you are not authorized by Event Handler Core System
+	 * @throws BadPayloadException when the payload couldn't be validated by Event Handler Core System 
+	 * @throws InvalidParameterException when the payload content couldn't be validated by Event Handler Core System
+	 * @throws ArrowheadException when internal server error happened at Event Handler Core System
+	 * @throws UnavailableServerException when Event Handler Core System is not available
+	 */
+	public void subscribeToEventHandler(final SubscriptionRequestDTO request) {
+
+		final CoreServiceUri uri = getCoreServiceUri(CoreSystemService.EVENT_SUBSCRIBE_SERVICE);
+		if (uri == null) {
+			logger.debug("Subscription couldn't be proceeded due to the following reason: " +  CoreSystemService.EVENT_SUBSCRIBE_SERVICE.name() + " not known by Arrowhead Context");
+			return;
+		}
+		
+		httpService.sendRequest( Utilities.createURI( getUriScheme(), uri.getAddress(), uri.getPort(), uri.getPath() ), HttpMethod.POST, Void.class, request );
```

Comment:
Remove unnecessary spaces after and before the brackets

Reply:
fixed